api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -83,6 +83,25 @@ def from_orm(cls, stock: Stock) -> "OfferStockResponse":
         return super().from_orm(stock)
 
 
+class OfferStockWithPriceCategoryResponse(OfferStockResponse):
+    priceCategoryLabel: str | None
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Ici simplement ajouter ce champ à la classe `OfferStockResponse` existante doit suffire (comme dit Damien au-dessus)

La réponse de la route va passer de : 
```js
{
  activationCode
  beginningDatetime,
  bookingLimitDatetime,
  cancellation_limit_datetime,
  id,
  is_forbidden_to_underage,
  isBookable,
  isExpired,
  isSoldOut,
  price,
} 
```

à : 
```js
{
  activationCode,
  beginningDatetime,
  bookingLimitDatetime,
  cancellation_limit_datetime,
  id,
  is_forbidden_to_underage,
  isBookable,
  isExpired,
  isSoldOut,
  price,
  priceCategoryLabel // <--- ici
} 
```

Et le front pourra gérer ça. 
Les nouvelles version de l'app auront quelque chose comme ` { priceCategoryLabel } = useGetOfferBlabla()`
Et les anciennes ignoreront le champ, parce qu'elle feront `{ isExpired, isSoldOut } = useGetOfferBlabla()`
(par exemple)